src/evaluator/matchers/all.ts

```diff
@@ -1,12 +1,11 @@
-import { logFactory } from '../../logger/sdkLogger';
-const log = logFactory('splitio-engine:matcher');
+// import { logFactory } from '../../logger/sdkLogger';
```

Comment:
**General comment for matchers**

Let's analyze this ones when polishing or trimming messages, but we might be good logging from the matcherFactory only the type of matcher and the result. 
We loose visibility though (on the specific attributes used for the matching), so might be good to keep until we can have a more verbose build without affecting the main one. Or we can just leave a code comment and get back to it when we work on the verbose/debug build